### PR TITLE
bracket model

### PR DESCRIPTION
**Pull request by JermaineKee**
bracket model ready for review

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
+
 # Create your models here.
+
+
+class Bracket(models.Model):
+    bracket_id = models.CharField(max_length=100)
+    title = models.CharField(max_length=100)
```
Comment (gmhurley) [inline api/models.py]: Let's make the length 255.

File: api/models.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+    start_dt = models.DateField(blank=True, null=True)
```
Comment (gmhurley) [inline api/models.py]: I think we'd want to use the date time field for these.

File: api/models.py [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+    end_dt = models.DateField(blank=True, null=True)
```
Comment (gmhurley) [inline api/models.py]: Let's use the date time field.

File: api/models.py [hunk 4]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
+
 # Create your models here.
+
+
+class Bracket(models.Model):
+    bracket_id = models.CharField(max_length=100)
```
Comment (gmhurley) [inline api/models.py]: We should get the bracket id by default. I don't believe that we need to define it here.

**Comment by gmhurley** [comment]
Make sure that you run the migrations on your machine, and get the migrations file checked into the repo.

**Comment by JermaineKee** [comment]
fixed the DateTimeField, and removed bracket_id.  migrated, but says no migrations to apply

File: api/models.py [hunk 5]
```diff
@@ -1,3 +1,10 @@
 from django.db import models
 
+
 # Create your models here.
+
+
+class Bracket(models.Model):
+    title = models.CharField(max_length=100)
```
Comment (gmhurley) [inline api/models.py]: Can we make the CharField 255? That seems to be sort of a standard for charfields that I've seen in class.

**Comment by gmhurley** [comment]
Don't forget to add the migration file.

**Comment by gmhurley** [comment]
:+1:

**Pull request merged**